app/konnect/analytics/custom-reports/use-cases.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Use Cases
+content_type: tutorial
+badge: plus
+---
+
+Custom reporting in {{site.konnect_saas}} gives you the power to monitor your API data in detail and export that data to CSV. 
+
+This tutorial guides you through the process of creating custom reports for key API metrics using {{site.konnect_saas}} custom reports. 
+
+You have just joined an organization as an API product manager. Your first task is to create a few reports that model business KPIs so that the executive team and investors have a grasp on the state of the API. You've selected the following KPIs because they would give a brief but wholistic understanding of the API: 
```

Comment:
```suggestion
In this tutorial scenario, you just joined an organization as an API product manager. Your first task is to create a few reports that model business KPIs so that the executive team and investors have a grasp on the state of the API. You've selected the following KPIs because they would give a brief, but wholistic, understanding of the API: 
```